### PR TITLE
feat: Adds the ability to  disable the "new message notification" popup

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR allows disabling the new message notification popup from the `chatwootSettings` config by adding `showNotificationPopup: false`. By default, the new message notification will be enabled.

Fixes https://linear.app/chatwoot/issue/CW-2834/is-there-a-way-to-disable-the-new-message-notification-popup

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

**Loom video**
https://www.loom.com/share/6117020ea1b44166b77066ee69ffff53?sid=07138d1f-82c3-4e5e-b45b-2601b83ac5f4

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/packs/sdk.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ const runSDK = ({ baseUrl, websiteToken }) => {
     type: getBubbleView(chatwootSettings.type),
     launcherTitle: chatwootSettings.launcherTitle || '',
     showPopoutButton: chatwootSettings.showPopoutButton || false,
+    showNotificationPopup: chatwootSettings.showNotificationPopup ?? true,
```
Comment (muhsin-k) [inline app/javascript/packs/sdk.js:65]: Could we use a more descriptive name such as showUnreadMessagesPopup?
cc @pranavrajs @scmmishra

**Review by muhsin-k — APPROVED** [review]
The code looks good to me except one minor comment. I have also verified it locally.
@iamsivin Please update the SDK docs after the merging the PR.

File: app/javascript/packs/sdk.js (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/packs/sdk.js]: showUnreadMessagesDialog

Comment (pranavrajs) [reply app/javascript/packs/sdk.js]: or showUnreadMessagesNotification

Comment (iamsivin) [reply app/javascript/packs/sdk.js]: @pranavrajs @muhsin-k Done https://github.com/chatwoot/chatwoot/pull/9594/commits/b0a1fff426652c5c44f327e1a8e2f8169c870ebc

**Review by pranavrajs — APPROVED** [review]
Code looks fine, please merge it after testing.

**Comment by iamsivin** [comment]
> Code looks fine, please merge it after testing.

@pranavrajs It's already tested by @muhsin-k

**Pull request merged**